package.json

```diff
@@ -70,6 +70,7 @@
     "ci-info": "^3.3.2",
     "date-fns": "^2.29.1",
     "ember-template-recast": "^6.1.3",
+    "eslint-formatter-kakoune": "Delapouite/eslint-formatter-kakoune#01f7697",
```

Comment:
What's the fork for?

Reply:
Its not a fork, its the upstream repo.

Reply:
I dont think it will be published as the repo is 6 years old. I will open an issue and ask if they plan on releasing a package.

Reply:
Should we just copy this [file](https://github.com/Delapouite/eslint-formatter-kakoune/blob/master/index.js) in with a comment about where it came from? Normally I wouldn't want to copy in external code like that but it seems like this could be better or safer than using a commit as a dependency?

Reply:
Probably, i think in this case it makes more sense to do that. I'll copy the file as well as test it again.

Reply:
With eslint, you can pass a repostitory to `--formater ./eslint-formatter-kakoune`. Do you know why template lint uses a different object format than eslint? This patch wouldn't be needed.

Reply:
Formatter has been released, updated package.json.